src/main/java/frc/robot/commands/ElevatorDefaultCommand.java

```diff
@@ -33,31 +36,39 @@ public void initialize() {
 
     @Override    
     public void execute(){
-        ChassisSpeeds chassisSpeeds = s_Swerve.getEstimatedFieldRelativeSpeeds();
-        
-        SmartDashboard.putNumber("xSpeed", chassisSpeeds.vxMetersPerSecond);
-        SmartDashboard.putNumber("ySpeed", chassisSpeeds.vyMetersPerSecond);
+    
+        Pose2d botPose = s_Swerve.getLimelightBotPose();
 
-        if (DriverStation.getAlliance().get().equals(Alliance.Red)) {
-            speakerCoordinate = new Pair<Double, Double>(8.0, 1.5);
-        } else {
-            speakerCoordinate = new Pair<Double, Double>(-8.0, 1.5);
-        }
+        boolean isRed = DriverStation.getAlliance().get().equals(Alliance.Red) ? true : false;
```

Comment:
This doesn't need to be a ternary, it can just be: 
boolean isRed = DriverStation.getAlliance().get().equals(Alliance.Red);